common-docs/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 * [FAQ](/faq)
 * [Projects](/projects)
 
+
```

Comment:
changes in this file look bogus. I'd revert those.

Reply:
All of this file?